charts/topolvm/README.md

```diff
@@ -99,7 +99,7 @@ You need to configure kube-scheduler to use topolvm-scheduler extender by referr
 | controller.prometheus.podMonitor.scrapeTimeout | string | `""` | Scrape timeout. If not set, the Prometheus default scrape timeout is used. |
 | controller.replicaCount | int | `2` | Number of replicas for CSI controller service. |
 | controller.securityContext.enabled | bool | `true` | Enable securityContext. |
-| controller.storageCapacityTracking.enabled | bool | `false` | Enable Storage Capacity Tracking for csi-provisoner. |
+| controller.storageCapacityTracking.enabled | bool | `false` | Enable Storage Capacity Tracking for csi-provisioner. |
```

Comment:
This README is auto-generated file.
Please follow [this instruction](https://github.com/topolvm/topolvm/blob/main/charts/topolvm/README.md#update-readme) to update it.
(Edit `README.md.gotmpl` and run helm-docs command.)

Reply:
I have already re-generated the README with `helm-docs`.

Since my corrections only affect the `values.yaml` and that part is automatically added during `helm-docs` I don´t think I need to update the `README.md.gotmpl`, right?

Reply:
Sorry, that is correct.